Lombiq.OrchardCoreApiClient/Constants/Timezones.cs

```diff
@@ -6,15 +6,10 @@ namespace Lombiq.OrchardCoreApiClient.Constants;
 
 public static class Timezones
 {
-    public static IList<string> GetTimezoneIds()
-    {
-        var nodaTimezones = new List<string>();
+    public static IList<string> TimezoneIds => GetTimeZoneIds();
```

Comment:
This is just proxying the method call but is not caching. Having just a method is good, because properties have the semantics of being quick to read, which is not necessarily the case here. But you can still cache to a static field.